src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1343,6 +1344,36 @@ def WriteGroupAttribute(
         # An empty list is the expected return for sending group write attribute.
         return []
 
+    def PrepareToReceiveBdxData(self) -> asyncio.Future:
```

Comment:
background - we're trying to clean up some of the python APIs to make it clearer what production controllers are and are not supposed to control. We SHOULD push some of the test-only stuff into a subclass, but that's a bit of a longer term project. In the meanwhile, if this is only for test, can you prefix it with TestOnly?